src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Api::V1::CookbookVersionsController < Api::V1Controller
+  before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation]
+  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation]
+  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation]
```

Comment:
Nice rollup. Sets things up nicely for more refactoring (mild renovation?) in the future for what will be a growing number of evaluation endpoints.